cmd/bridge-auth-delete.go

```diff
@@ -0,0 +1,72 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/0chain/gosdk/zcncore"
+	"github.com/spf13/cobra"
+	"log"
+	"strconv"
+	"strings"
+	"sync"
+)
+
+var deleteAuthorizerConfigCmd = &cobra.Command{
+	Use:   "bridge-auth-delete",
+	Short: "Delete ZCNSC authorizer by ID",
+	Long:  `Delete ZCNSC authorizer by ID`,
+	Args:  cobra.MinimumNArgs(0),
+	Run: func(cmd *cobra.Command, args []string) {
+
+		var (
+			err error
+		)
+
+		payload := &zcncore.DeleteAuthorizerPayload{}
```

Comment:
we need authorizer id here, since txn will be executed with delegate wallet

Reply:
Updated